control-plane/pkg/reconciler/broker/namespaced_controller.go

```diff
@@ -212,3 +224,15 @@ func NewNamespacedController(ctx context.Context, watcher configmap.Watcher, env
 
 	return impl
 }
+
+func (r *NamespacedReconciler) getCaCerts() (string, error) {
+	secret, err := r.SecretLister.Secrets(r.SystemNamespace).Get(brokerIngressTLSSecretName)
+	if err != nil {
+		return "", fmt.Errorf("failed to get CA certs from %s/%s: %w", r.SystemNamespace, brokerIngressTLSSecretName, err)
+	}
+	caCerts, ok := secret.Data[caCertsSecretKey]
+	if !ok {
+		return "", fmt.Errorf("failed to get CA certs from %s/%s: missing %s key", r.SystemNamespace, brokerIngressTLSSecretName, caCertsSecretKey)
+	}
+	return string(caCerts), nil
+}
```

Comment:
Same as above, I'd leave out the TLS for namespaced broker until we have a design that would work for it